mambaSharedFramework/Pantos-Generic HLS Playlist Parsing/PantosTag.swift

```diff
@@ -371,7 +381,10 @@ extension PantosTag: HLSTagDescriptor, Equatable {
         case .EXT_X_DISCONTINUITY_SEQUENCE:
             return GenericSingleTagValidator<Int>(tag: pantostag,
                                                   singleValueIdentifier:PantosValue.discontinuitySequence)
-            
+        case .EXT_X_BITRATE:
+            return GenericSingleTagValidator<Double>(tag: pantostag,
+                                                  singleValueIdentifier:PantosValue.bandwidthBPS)
```

Comment:
Minor: You can format code the way Xcode thinks is correct by selecting lines and Ctrl-I.

That will get rid of the mismatch between `tag:` and `singleValueIdentifier:`

Reply:
Fixed